src/views/shared/SecurityBeacon.js

```diff
@@ -15,6 +15,18 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
   var _ = Okta._,
       $ = Okta.$;
 
+  // TODO(OKTA-118980): Server should send description as part of appState.
+  function getSecurityImageDescriptionFromUrl(imgSrc) {
+    var parser = document.createElement('a');
+    parser.href = imgSrc;
+
+    // Get stuff between the last slash, and the first period after that slash.
+    // (if the string ends with a slash, use the second-to-last slash instead.)
+    var match = /^.*\/([^.]+).*\/?$/.exec(parser.pathname);
+    var basename = (match.length >= 2) ? match[1] : null;
+    return basename ? (basename + ' security image') : '';
```

Comment:
I believe i18n for this is going to require several steps. As you said, we need the properties. And then we need to pipe them through to the client. It looks like SecurityImage can hold alt text, though I'm not sure whether that actually gets populated anywhere (probably not?). SessionCookieManager uses UserSecurityFields, which holds the security image as a string, and is ultimately populated by the Login table which also stores "pwdImg" as a string. At least getimage.jsp should be easy to update once all that is changed.